pkg/features/embedded/elemental-rootfs/etc/dracut.conf.d/02-elemental-rootfs.conf

```diff
@@ -1,2 +1 @@
-install_items+=" /usr/lib/systemd/system/elemental-rootfs.service /etc/systemd/system/initrd-fs.target.requires/elemental-rootfs.service "
```

Comment:
This is redundant with the lines from https://github.com/rancher/elemental-toolkit/blob/48966fa3e341d0b8b036fde7c27f3ea55d9cc747/pkg/features/embedded/elemental-rootfs/usr/lib/dracut/modules.d/30elemental-rootfs/module-setup.sh#L36-L40

Reply:
Yep, I think it was just a copy-paste that stuck around from the `elemental-setup` feature